sdks/python/apache_beam/io/aws/sqsio.py

```diff
@@ -0,0 +1,74 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import apache_beam as beam
+from apache_beam.transforms.external import BeamJarExpansionService
+from apache_beam.transforms.external import SchemaAwareExternalTransform
+
+__all__ = ['ReadFromSQS']
+
+
+class ReadFromSQS(beam.PTransform):
+  """Reads messages from an AWS SQS queue.
+    
+    Returns a PCollection of Beam Rows, each representing a SqsMessage object.
+    This transform uses the Java SDK implementation, and most of AWS related 
+    connectivity configurations can be set through a custom PipelineOptions 
+    object with the same properties seen on the AwsOptions for Java SDK. 
+    For more information see: 
+    https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/aws/sqs/SqsIO.html
+    """
+  URN = "beam:schematransform:org.apache.beam:aws:sqs_read:v1"
+
+  def __init__(
```

Comment:
(question to @ahmedabu98) Shall we add this external transform to auto wrapper?

Reply:
quick question, can the integration test use the newly generated transform instead of the ExternalTransformProvider? given the dependencies the new Python wrapper should already be in place, is that right?

Reply:
Ok, I made the changes to make the IT use the ExternalTransformProvider. Thanks for for the pointers.

Reply:
I think it depends -- some test environments won't set up the SDK before testing, i.e. it won't run the code that generates the wrapper file. So it's safer to use ExternalTransformProvider

Thanks for making those changes